docs/diagrams/LogicClassDiagram.puml

```diff
@@ -18,9 +18,9 @@ Class Prefix
 }
 
 package Command {
-Class XYZCommand
-Class CommandResult
-Class "{abstract}\nCommand" as Command
+    Class XYZCommand
+    Class CommandResult
+    Class "{abstract}\nCommand" as Command
```

Comment:
Can you change this file back to the original version or leave this file out?

Reply:
oops sry.